src/incidence_list.jl

```diff
@@ -110,6 +110,41 @@ function add_vertex!(g::GenericIncidenceList{V,E}, v::V) where {V,E}
   add_vertex!(g.vertices, g.inclist, v)
 end
 
+function delete_vertex!(vertices::Dict{Int, V}, inclist::Dict{Int, E}, v::V, outnei) where {V,E}
+  nedges = 0
+  # delete all connected edges
+  for vid in union(map(x->x.index, outnei), v.index)
+    count = 0
+    for ed in inclist[vid]
+      count += 1
+      if ed.source.index == v.index || ed.target.index == v.index
+        # delete this edge
+        # @show count, inclist[vid]
+        deleteat!(inclist[vid], count)
+        count -= 1
+        nedges += 1
+      end
+    end
+    delete!(inclist, vid)
+  end
+
+  delete!(vertices, v.index)
+  return nedges
+end
+
+function delete_vertex!(g::GenericIncidenceList{V,E}, v::V) where {V,E}
```

Comment:
Suggestion - standard pattern would be delete_vertex(v::V, g::GenericIncidenceList{V,E}) where {V,E})

Maybe change to match standard?

Reply:
done